Frameworks/UIKit/UIButton.mm

```diff
@@ -552,8 +552,7 @@ - (void)setTitleColor:(UIColor*)color forState:(UIControlState)state {
     // ConvertUIColorToWUColor:nil creates a valid WUColor with null comObj
     // which isn't what we want
     if (color) {
-        WUColor* convertedColor = XamlUtilities::ConvertUIColorToWUColor(color);
-        WUXMSolidColorBrush* titleColorBrush = [WUXMSolidColorBrush makeInstanceWithColor:convertedColor];
+        WUXMSolidColorBrush* titleColorBrush = [[WUXMSolidColorBrush makeInstanceWithColor:XamlUtilities::ConvertUIColorToWUColor(color)] autorelease];
```

Comment:
good question; i'll give it a shot

---
In reply to: [97843932](https://github.com/Microsoft/WinObjC/pull/1803#discussion_r97843932) [](ancestors = 97843932)